website/www/site/layouts/index.html

```diff
@@ -60,6 +60,24 @@ <h2 class="pillars-title">
   </div>
 {{ end }}
 
+{{ define "playground-section" }}
+<div class="body__contained body__section-nav">
+  <h1>Try Beam Playground</h1>
+  <p>Beam Playground is an interactive environment to try out Beam transforms and examples without having to install Apache Beam in your environment.
+    You can try an Apache Beam examples at <a href="https://frontend-beta-dot-apache-beam-testing.appspot.com/">Beam Playground (Beta)</a>.
```

Comment:
```suggestion
    You can try an Apache Beam examples at <a href="http://play.beam.apache.org/">Beam Playground (Beta)</a>.
```

Let's use the proper  url right?

Reply:
changed